integr8ly/integreatly-operator
# Mgdapi 2446

# Description

Adds the integration of the new 3scale tenant with RHSSO

## Verification steps

1. Install RHOAM with the multi tenant install type
`INSTALLATION_TYPE=multitenant-managed-api make cluster/prepare/local && INSTALLATION_TYPE=multitenant-managed-api make code/run`
2. Install the dev sandbox on the cluster and hook up RHSSO 
3. Register to the cluster via the dev sandbox
4. Verify if the new 3scale account is create for the new user with the auth provider configured
